src/resampler.ts

```diff
@@ -269,29 +258,27 @@ class Resampler {
       throw new Error(`Invalid inputFrame type: ${typeof inputFrame}. Expected Float32Array or Int16Array.`);
     }
 
-    try {
-      this._memoryBuffer.set(
-        inputBuffer,
-        this._inputBufferAddress / Int16Array.BYTES_PER_ELEMENT,
-      );
+    const memoryBuffer = new Int16Array(this._wasmMemory.buffer);
+    const memoryBufferView = new DataView(this._wasmMemory.buffer);
```

Comment:
Can't you wait to create `memoryBufferView` until after the `_pvResamplerProcess` call?